### PR TITLE
Apr 1989 pmo outcome page additions

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.AdminService.Web/wwwroot/stylesheets/application.css [hunk 1]
```diff
@@ -1657,3 +1657,49 @@ TODO: change to 'das-page-navigation' and BEM
...
+
+.das-button--inline-link:active {
+  color: #0b0c0c;
+  background-color: #ffbf47;
+}
+
+.das-button--inline-link:focus {
+  color: #0b0c0c;
+  background-color: #ffbf47;
+}
```
Comment (martin-hornsby) [inline src/SFA.DAS.AdminService.Web/wwwroot/stylesheets/application.css:1705]: Are these to be separate from those in lines 1679 above?

File: src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Financial/Application_ReadOnly.cshtml [hunk 2]
```diff
@@ -102,68 +98,89 @@
...
+                            Financial health assessed on
                         </dt>
-                    }
-                    <dd class="govuk-summary-list__value">
-                            @Model.FinancialReviewDetails.Comments
+                        <dd class="govuk-summary-list__value">
+                            @if (Model.FinancialReviewDetails.GradedDateTime.HasValue)
+                            {
+                                @Html.Raw(Model.FinancialReviewDetails.GradedDateTime.Value.ToLongDateString())
+                            }
```
Comment (martin-hornsby) [inline src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Financial/Application_ReadOnly.cshtml:161]: Just noticed this but shouldn't this be: `@Model.FinancialReviewDetails.GradedDateTime.ToSfaShortDateString()` ?

May need to include: `@using SFA.DAS.AdminService.Common.Extensions`

Currently, when I use American English, this displays differently 😈

File: src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpFinancialController.cs [hunk 3]
```diff
@@ -157,13 +157,18 @@ public async Task<IActionResult> GradeApplication(Guid applicationId, [FromForm]
         [HttpPost("/Roatp/Financial/Clarification/{applicationId}")]
         public async Task<IActionResult> SubmitClarification(Guid applicationId, RoatpFinancialClarificationViewModel vm)
         {
```
Comment (martin-hornsby) [inline src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpFinancialController.cs:159]: Just a comment to think about... this function is getting chunky and a little difficult to follow, could it be refactored into 2 or 3 smaller private methods?

**Review by martin-hornsby — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpFinancialController.cs (hunk 3 above)
Comment (MarkFCain) [reply src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpFinancialController.cs:159]: yes, good idea

File: src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Financial/Application_ReadOnly.cshtml (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Financial/Application_ReadOnly.cshtml:161]: not sure, I'll try it

File: src/SFA.DAS.AdminService.Web/wwwroot/stylesheets/application.css (hunk 1 above)
Comment (gdbarnes) [reply src/SFA.DAS.AdminService.Web/wwwroot/stylesheets/application.css:1705]: No reason for them to be. I'll put them into one rule.

File: src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Financial/Application_ReadOnly.cshtml (hunk 2 above)
Comment (MarkFCain) [reply src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Financial/Application_ReadOnly.cshtml:161]: yes, will do

**Review by martin-hornsby — APPROVED** [review]

**Review by martin-hornsby — APPROVED** [review]

**Pull request merged**